src/App.js

```diff
@@ -58,6 +58,35 @@ const App = () => {
     setSnoozeList(updatedSnoozeList)
   }
 
+  const handleUpdateSnooze = async ({ notifyDate, snooze }) => {
+    setErrorMessage('')
```

Comment:
I would put this at the end of this function, in order to avoid a double set invocation on line 65 and 72

Reply:
Done